src/editor/Model/WindowTree.re

```diff
@@ -1,3 +1,5 @@
+open Oni_Core;
```

Comment:
I was not too sure if we want expose everything from `OniCore` here or do something like `module Utility = Oni_Core__Utility` and then prefix `last` and `dropLast`.

Reply:
Good question! I've been using the `open Oni_Core` style a lot, but I'm not sure its canonical Reason / OCaml (which prefer less pollution in the module as far as I can tell). This is reasonable for now though and consistent with the current styling of the code 👍